third_party/airflow/examples/bad_armada.py

```diff
@@ -0,0 +1,136 @@
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from armada.operators.armada import ArmadaOperator
+
+from armada_client.k8s.io.api.core.v1 import generated_pb2 as core_v1
+from armada_client.k8s.io.apimachinery.pkg.api.resource import (
+    generated_pb2 as api_resource,
+)
+
+from armada_client.armada import (
+    submit_pb2,
+)
+
+from armada_client.client import ArmadaClient
+import grpc
+
+import pendulum
+
+from armada.operators.jobservice import JobServiceClient
+
+
+def submit_sleep_job():
+    """
+    Simple armada job
+    """
+    pod = core_v1.PodSpec(
+        containers=[
+            core_v1.Container(
+                name="sleep",
+                image="busybox",
+                args=["sleep", "10s"],
+                securityContext=core_v1.SecurityContext(runAsUser=1000),
+                resources=core_v1.ResourceRequirements(
+                    requests={
+                        "cpu": api_resource.Quantity(string="120m"),
+                        "memory": api_resource.Quantity(string="510Mi"),
+                    },
+                    limits={
+                        "cpu": api_resource.Quantity(string="120m"),
+                        "memory": api_resource.Quantity(string="510Mi"),
+                    },
+                ),
+            )
+        ],
+    )
+
+    return [submit_pb2.JobSubmitRequestItem(priority=1, pod_spec=pod)]
+
+
+def submit_bad_job():
```

Comment:
I am not sure how termperary this file is, but having a lot of repeatable code is not ideal for an example. Maybe make creating a dummy pod spec a function with args that could mkae ifakail or not fail?